web/src/main/webapp/libs/app.bundle.min.js

```diff
@@ -2901,7 +2901,12 @@
                 return e = e.map(e => e.replace(/[-_]+/g, "")), e.length > 0 ? e : void 0
             },
             name: "customNavigatorDetector"
-        };
+        },
+        ofmeet = {
```

Comment:
Default value (en) for lang/ofmeet-xx.
It is better to read from a file, but Jitsi's language file is also embedded (at the time of build because it uses require), so it is hard-coded.
I could use synchronous XHR to get around this, but synchronous XHR is currently discouraged, so I decided not to.